sklearn/utils/validation.py

```diff
@@ -519,7 +519,13 @@ def column_or_1d(y, warn=False):
     y : array-like
 
     warn : boolean, default False
-       To control display of warnings.
+        To control display of warnings.
+
+    copy: boolean, default False
+        Whether or not to copy y.
+
+    order: 'C' or 'F', default 'C'
```

Comment:
I would leave the default to `None` meaning that we accept non-contiguous arrays as input by default.